### PR TITLE
Letter 't' from #436

### PR DESCRIPTION
**Pull request by luzpaz**
Continuing to separate ~#428~ #436 in to separate PRs for each letter. Also removed/corrected some entries that were not relevant/precise and augmented others. This PR has been sorted as well. 

Please review closely, some of the entries still may be incorrect or may need refinement.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10500,41 +10590,60 @@ transmiting->transmitting
...
+transpeorted->transported
 transporation->transportation
+transportatin->transportation
 transtion->transition
 transtions->transitions
 transtition->transition
 transtitions->transitions
 transtorm->transform
 transtormed->transformed
+transverse->traverse
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: "transverse" is a word (e.g., "transverse axis")

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10500,41 +10590,60 @@ transmiting->transmitting
...
+trigerred->triggered
 trigerring->triggering
 trigers->triggers
 triggerd->triggered
+triggeres->triggers
 triggerred->triggered
 triguered->triggered
+trimed->trimmed
+tring->trying
+trings->strings
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "rings" based on t/r proximity on QWERTY

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10500,41 +10590,60 @@ transmiting->transmitting
...
 triangultaion->triangulation
+trigerred->triggered
 trigerring->triggering
 trigers->triggers
 triggerd->triggered
+triggeres->triggers
 triggerred->triggered
 triguered->triggered
+trimed->trimmed
+tring->trying
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "ring"

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: good point. added:
```
tring->trying, string, 
trings->strings, rings,
```

**Pull request merged**